packages/platform/types/bookings.ts

```diff
@@ -0,0 +1,42 @@
+import { Type } from "class-transformer";
+import { ValidateNested, IsNumber, Min, Max, IsOptional, IsArray, IsEnum } from "class-validator";
+
+enum Status {
```

Comment:
In Prisma schema we have enum used for Booking.status:

```
enum BookingStatus {
  CANCELLED     @map("cancelled")
  ACCEPTED      @map("accepted")
  REJECTED      @map("rejected")
  PENDING       @map("pending")
  AWAITING_HOST @map("awaiting_host")
}
```

Can we re-use it? Also, it does not have "upcoming" and "recurring" like in enum here, so I am wondering how does the mis-match work?

Reply:
@supalarry honestly idk why the booking status varies in our prisma schema and the filters object, I'm guessing we transform the data in a certain way in the webapp right and that's why we can'e reuse it other places